WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsListTopBanner.swift

```diff
@@ -0,0 +1,90 @@
+import Foundation
+import SwiftUI
+
+/// Banner to inform the users how custom fields are saved.
+///
+struct CustomFieldsListTopBanner: UIViewRepresentable {
```

Comment:
Based on other implementations in the app, I think this can be made reusable with some tweaks, but given my lack of experience with SwiftUI and that we are trying to get this ready ASAP, I decided to follow the same approach as the other screens.

Reply:
I think this is OK, we can always refactor it if/when there's a need to use it in a new place.